### PR TITLE
add saucectlVersion to the metadata

### PR DESCRIPTION
**Pull request by valentyn88**

### CODE REVIEW

File: src/testcafe-runner.js [hunk 1]
```diff
@@ -190,7 +192,7 @@ async function run (runCfgPath, suiteName, timeoutSec) {
   }
 
   const region = cfg.runCfg.sauce.region || 'us-west-1';
-  await runReporter({ assetsPath: cfg.assetsPath, region, metadata: cfg.metadata, ...testCafeResults });
+  await runReporter({ assetsPath: cfg.assetsPath, region, metadata: cfg.metadata, saucectlVersion: cfg.saucectlVersion, ...testCafeResults });
```
Comment (FriggaHel) [inline src/testcafe-runner.js:195]: Shouldn't this be that ? 

```suggestion
  await runReporter({ assetsPath: cfg.assetsPath, region, metadata: cfg.metadata, saucectlVersion: cfg.runCfg.saucectlVersion, ...testCafeResults });
```

(Considering `L192` as a reference)

Comment (valentyn88) [reply src/testcafe-runner.js:195]: I tested it locally, it should be as in my commit.

**Comment by alexplischke** [comment]
Why was this merged when https://github.com/saucelabs/saucectl/pull/323 is going to pass it via env vars?

**Comment by valentyn88** [comment]
> Why was this merged when [saucelabs/saucectl#323](https://github.com/saucelabs/saucectl/pull/323) is going to pass it via env vars?

I didn't pay attention that it was not approved, because other PR's were approved. I didn't want to revert changes and almost prepared all branches where I use env var.

**Pull request merged**